Flaky CreateAppEngineFlexWtpProjectTest

```
!MESSAGE IWAE0027E Could not retrieve the validation configuration for resource testproject0.9103036092156647.
```

On a Kokoro Windows build. Maybe some validation-related job was running in parallel? The stack trace does not involve our code, so this could be tricky to debug.

Full stack trace:
```
Running com.google.cloud.tools.eclipse.appengine.newproject.flex.CreateAppEngineFlexWtpProjectTest

!ENTRY org.eclipse.wst.validation 4 0 2017-05-13 08:54:55.920
!MESSAGE Resource '/testproject0.9103036092156647' does not exist.
!STACK 1
org.eclipse.core.internal.resources.ResourceException: Resource '/testproject0.9103036092156647' does not exist.
	at org.eclipse.core.internal.resources.Resource.checkExists(Resource.java:335)
	at org.eclipse.core.internal.resources.Resource.checkAccessible(Resource.java:209)
	at org.eclipse.core.internal.resources.Project.checkAccessible(Project.java:141)
	at org.eclipse.core.internal.resources.Resource.accept(Resource.java:60)
	at org.eclipse.core.internal.resources.Resource.accept(Resource.java:52)
	at org.eclipse.wst.validation.internal.ValBuilderJob.fullBuild(ValBuilderJob.java:216)
	at org.eclipse.wst.validation.internal.ValBuilderJob.run(ValBuilderJob.java:178)
	at org.eclipse.wst.validation.internal.ValBuilderJob.runInWorkspace(ValBuilderJob.java:126)
	at org.eclipse.core.internal.resources.InternalWorkspaceJob.run(InternalWorkspaceJob.java:39)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:56)
!SUBENTRY 1 org.eclipse.core.resources 4 368 2017-05-13 08:54:55.936
!MESSAGE Resource '/testproject0.9103036092156647' does not exist.

!ENTRY org.eclipse.wst.validation 4 0 2017-05-13 08:54:56.045
!MESSAGE IWAE0027E Could not retrieve the validation configuration for resource testproject0.9103036092156647.
!STACK 0
java.lang.reflect.InvocationTargetException: IWAE0027E Could not retrieve the validation configuration for resource testproject0.9103036092156647.
	at org.eclipse.wst.validation.internal.ConfigurationManager.getProjectConfiguration(ConfigurationManager.java:135)
	at org.eclipse.wst.validation.internal.ConfigurationManager.getConfiguration(ConfigurationManager.java:251)
	at org.eclipse.wst.validation.internal.ValManager.getValidatorsNotCached(ValManager.java:226)
	at org.eclipse.wst.validation.internal.ValManager.access$1(ValManager.java:220)
	at org.eclipse.wst.validation.internal.ValManager$ValidatorCache.getValidatorsCached(ValManager.java:1263)
	at org.eclipse.wst.validation.internal.ValManager.getValidators(ValManager.java:197)
	at org.eclipse.wst.validation.internal.ValManager.accept(ValManager.java:745)
	at org.eclipse.wst.validation.internal.ValBuilderJob.run(ValBuilderJob.java:201)
	at org.eclipse.wst.validation.internal.ValBuilderJob.runInWorkspace(ValBuilderJob.java:126)
	at org.eclipse.core.internal.resources.InternalWorkspaceJob.run(InternalWorkspaceJob.java:39)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:56)
Caused by: org.eclipse.core.internal.resources.ResourceException: Resource '/testproject0.9103036092156647' does not exist.
	at org.eclipse.core.internal.resources.Resource.checkExists(Resource.java:335)
	at org.eclipse.core.internal.resources.Resource.checkAccessible(Resource.java:209)
	at org.eclipse.core.internal.resources.Project.checkAccessible(Project.java:141)
	at org.eclipse.core.internal.resources.Resource.checkAccessibleAndLocal(Resource.java:215)
	at org.eclipse.core.internal.resources.Resource.getSessionProperty(Resource.java:1181)
	at org.eclipse.wst.validation.internal.ConfigurationManager.getProjectConfiguration(ConfigurationManager.java:126)
	... 10 more
Root exception:
org.eclipse.core.internal.resources.ResourceException: Resource '/testproject0.9103036092156647' does not exist.
	at org.eclipse.core.internal.resources.Resource.checkExists(Resource.java:335)
	at org.eclipse.core.internal.resources.Resource.checkAccessible(Resource.java:209)
	at org.eclipse.core.internal.resources.Project.checkAccessible(Project.java:141)
	at org.eclipse.core.internal.resources.Resource.checkAccessibleAndLocal(Resource.java:215)
	at org.eclipse.core.internal.resources.Resource.getSessionProperty(Resource.java:1181)
	at org.eclipse.wst.validation.internal.ConfigurationManager.getProjectConfiguration(ConfigurationManager.java:126)
	at org.eclipse.wst.validation.internal.ConfigurationManager.getConfiguration(ConfigurationManager.java:251)
	at org.eclipse.wst.validation.internal.ValManager.getValidatorsNotCached(ValManager.java:226)
	at org.eclipse.wst.validation.internal.ValManager.access$1(ValManager.java:220)
	at org.eclipse.wst.validation.internal.ValManager$ValidatorCache.getValidatorsCached(ValManager.java:1263)
	at org.eclipse.wst.validation.internal.ValManager.getValidators(ValManager.java:197)
	at org.eclipse.wst.validation.internal.ValManager.accept(ValManager.java:745)
	at org.eclipse.wst.validation.internal.ValBuilderJob.run(ValBuilderJob.java:201)
	at org.eclipse.wst.validation.internal.ValBuilderJob.runInWorkspace(ValBuilderJob.java:126)
	at org.eclipse.core.internal.resources.InternalWorkspaceJob.run(InternalWorkspaceJob.java:39)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:56)
[ERROR] Timeout 600 s exceeded. Process was killed.
```

--- a/plugins/com.google.cloud.tools.eclipse.appengine.newproject.test/src/com/google/cloud/tools/eclipse/appengine/newproject/flex/CreateAppEngineFlexWtpProjectTest.java
+++ b/plugins/com.google.cloud.tools.eclipse.appengine.newproject.test/src/com/google/cloud/tools/eclipse/appengine/newproject/flex/CreateAppEngineFlexWtpProjectTest.java
@@ -26,6 +26,7 @@ import com.google.cloud.tools.eclipse.appengine.libraries.model.MavenCoordinates
 import com.google.cloud.tools.eclipse.appengine.libraries.repository.ILibraryRepositoryService;
 import com.google.cloud.tools.eclipse.appengine.newproject.AppEngineProjectConfig;
 import com.google.cloud.tools.eclipse.appengine.newproject.CreateAppEngineWtpProject;
+import com.google.cloud.tools.eclipse.test.util.project.ProjectUtils;
 import java.io.File;
 import java.io.IOException;
 import java.lang.reflect.InvocationTargetException;
@@ -75,6 +76,8 @@ public class CreateAppEngineFlexWtpProjectTest {
 
   @After
   public void tearDown() throws CoreException {
+    // https://github.com/GoogleCloudPlatform/google-cloud-eclipse/issues/1945
+    ProjectUtils.waitForProjects(project);
     project.delete(true, monitor);
   }
 

